btcjson/chainsvrresults.go

```diff
@@ -92,7 +92,7 @@ type SoftForkDescription struct {
 type Bip9SoftForkDescription struct {
 	Status    string `json:"status"`
 	Bit       uint8  `json:"bit"`
-	StartTime int64  `json:"startTime"`
+	StartTime int64  `json:"start_time"`
```

Comment:
Good question...it would appears that it does, unless the `json` library falls back to a fuzzy match on the field name within the struct itself?

Reply:
> does this break compat with older nodes?

It does...

> unless the json library falls back to a fuzzy match on the field name within the struct itself?

Doesn't seem to be the case unfortunately due to the `_`. How should we address this? This same way we did with `SoftForks` and `UnifiedSoftForks` in `GetBlockChainInfoResult`?

Reply:
We can. I'm wondering if this is even worth the hassle though, as it would only affect `bitcoind < 0.19.0` and it wouldn't affect `btcd` at all. Also there's no failure from unmarshaling in this case, the `StartTime` field just wouldn't be set.

Reply:
Here's an option: remove this field, and add two new fields `StartTime1` and `StartTime2`, then create a new method `StartTime` that returns the value. This breaks old clients, but gracefully (imo) as they'll likely see a compile error that they're passing in a method, when it's expecting an `int64`.

Reply:
Done.